src/pages/TestNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import { NotificationStack } from '@/components/Notification'
+import { NotificationType } from '@/libs/constants'
+import { useNotificationStore } from '@/stores/notification'
+
+const TestNotification = () => {
+  const [notifications, setNotifications] = useNotificationStore((state) => [
+    state.notifications,
+    state.setNotifications,
+  ])
+
+  const handleAddNotification = (type: NotificationType, message: string) => {
```

Comment:
This logic should be handle by Notification component by it own. Number of notification that show on screen must be a param. Create a hook (ex: useNotification) and export the addNofication function instead.
ex: `const { addNotification } = useNotification()`

Reply:
Please check my forth commit, thanks.